aggregator/src/util.rs

```diff
@@ -9,29 +13,28 @@ use snark_verifier::loader::halo2::halo2_ecc::halo2_base::{
     gates::{flex_gate::FlexGateConfig, GateInstructions},
     AssignedValue, Context,
 };
-
-use crate::{
-    aggregation::RlcConfig,
-    constants::{DIGEST_LEN, INPUT_LEN_PER_ROUND, MAX_AGG_SNARKS},
-    DEFAULT_KECCAK_ROWS, NUM_ROUNDS,
+use zkevm_circuits::keccak_circuit::keccak_packed_multi::{
+    get_num_rows_per_round, get_num_rows_per_update,
 };
 
-use std::env::var;
+// Calculates the maximum keccak updates (1 absorb, or 1 f-box invoke)
+// needed for the number of snarks
+pub(crate) fn get_max_keccak_updates(max_snarks: usize) -> usize {
```

Comment:
maybe add a few comments here why pi_rounds = **2** and why chunk_hash_rounds is **2** * max_snarks

Reply:
Comments added.